tools/flakiness_checker.py

```diff
@@ -28,67 +28,79 @@
 import re
 
 
-DEFAULT_NUM_TRIALS = 500
+DEFAULT_NUM_TRIALS = 10000
+DEFAULT_VERBOSITY = 2
+
+def run_test_trials(args):
+    test_path = args.test_path + ":" + args.test_name
+    print("testing: " + test_path)
 
```

Comment:
Try to use logging instead of print. This allows you to make log messages without having to concatenate strings